python/lsst/afw/table/schema/schemaContinued.py

```diff
@@ -0,0 +1,204 @@
+#
+# LSST Data Management System
+# Copyright 2017 LSST/AURA.
+#
+# This product includes software developed by the
+# LSST Project (http://www.lsst.org/).
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the LSST License Statement and
+# the GNU General Public License along with this program.  If not,
+# see <http://www.lsstcorp.org/LegalNotices/>.
+#
+from __future__ import absolute_import, division, print_function
+
+__all__ = ["Key", "Field", "SchemaItem"]
+
+from past.builtins import basestring
+from builtins import str as futurestr
+from future.utils import with_metaclass
+
+import numpy as np
+import fnmatch
+import re
+import collections
+import astropy.units
+
+from ...geom import Angle
```

Comment:
Hmm.  I agree it is a bit easier to read, but I don't think relative imports stop being a good idea within a package just because the number of levels increases.